caesarsCipher/script.js

```diff
@@ -0,0 +1,75 @@
+const form = document.getElementById('my-form');
+let userInput = document.getElementById('user-input');
+let result = document.getElementById('ciphered-word');
+const copyButton = document.getElementById('copy-btn');
+const copied = document.getElementById('copied');
+
+const TIMEOUT = 3000;
+
+function rot13(str) {
+    let rot13 = {
```

Comment:
Seems like Indian code to me 😂

Please look at Char Codes in javascript String class and String prototype for a more generic implementation.

Reply:
fixed